frog/imports/ui/GraphEditor/SidePanel/ActivityPanel.js

```diff
@@ -41,6 +41,7 @@ const EditClass = props => {
         <h3>
           <ChangeableText
             EditComponent={RenameField}
+            activityId={activity._id}
```

Comment:
I do not see `frog-utils/ChangeableText` using an `activityId` props, so I don't understand this change. Could you enlighten me?